### PR TITLE
Add ZKsync WebSocket support.

### PR DESCRIPTION
**Pull request by bgravenorst**
# Description

Documented WebSocket support for the ZKsync Era network.

## Preview

[(https://metamask-docs-git-zksync-websockets-metamask-web.vercel.app/services/reference/zksync/json-rpc-methods/)](https://metamask-docs-git-zksync-websockets-metamask-web.vercel.app/services/reference/zksync/json-rpc-methods/)

## Checklist

Complete this checklist before merging your PR:

- [x] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [x] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ of the [MetaMask developer page](https://metamask.io/developer/).
 
 ## July 2024
 
+- Documented [ZKsync Era WebSocket support](/services/reference/zksync). ([1438](https://github.com/MetaMask/metamask-docs/pull/1438))
```
Comment (joaniefromtheblock) [inline docs/whats-new.md:14]:
```suggestion
- Documented [ZKsync Era WebSocket support](/services/reference/zksync). ([# 1438](https://github.com/MetaMask/metamask-docs/pull/1438))
```

File: services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx [hunk 2]
```diff
@@ -1,15 +1,23 @@
-import Tabs from "@theme/Tabs"
-import TabItem from "@theme/TabItem"
+import Tabs from '@theme/Tabs';
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx:1]: @bgravenorst we have been using double quotes for our docs. Is that okay?

File: services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
   <TabItem value="cURL">
 
 ```bash
 curl https://zksync-mainnet.infura.io/v3/YOUR-API-KEY \
-  -X POST \
-  -H "Content-Type: application/json" \
-  -d '{"jsonrpc": "2.0", "method": "eth_accounts", "params": [], "id": 1}'
+    -X POST \
+    -H "Content-Type: application/json" \
+    -d '{"jsonrpc":"2.0","method":"eth_accounts","params":[],"id":1}'
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx:11]:
```suggestion
    -d '{"jsonrpc": "2.0", "method": "eth_accounts", "params": [], "id": 1}'
```

File: services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
+```
+
+  </TabItem>
+  <TabItem value="WSS">
+
+```bash
+wscat -c wss://zksync-mainnet.infura.io/ws/v3/YOUR-API-KEY -x '{"jsonrpc":"2.0","method":"eth_accounts","params":[],"id":1}'
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx:18]:
```suggestion
wscat -c wss://zksync-mainnet.infura.io/ws/v3/YOUR-API-KEY -x '{"jsonrpc": "2.0", "method": "eth_accounts", "params": [], "id": 1}'
```

File: services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx (hunk 2 above)
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx:1]:
```suggestion
import Tabs from "@theme/Tabs";
```

File: services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx [hunk 5]
```diff
@@ -1,15 +1,23 @@
-import Tabs from "@theme/Tabs"
-import TabItem from "@theme/TabItem"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx:2]:
```suggestion
import TabItem from "@theme/TabItem";
```

File: services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx [hunk 6]
```diff
@@ -1,15 +1,23 @@
-import Tabs from "@theme/Tabs"
-import TabItem from "@theme/TabItem"
+import Tabs from '@theme/Tabs';
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx:1]:
```suggestion
import Tabs from "@theme/Tabs";
```

File: services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx [hunk 7]
```diff
@@ -1,15 +1,23 @@
...
+import TabItem from '@theme/TabItem';
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx:2]:
```suggestion
import TabItem from "@theme/TabItem";
```

File: services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx [hunk 8]
```diff
@@ -1,15 +1,23 @@
...
   <TabItem value="cURL">
 
 ```bash
 curl https://zksync-mainnet.infura.io/v3/YOUR-API-KEY \
-  -X POST \
-  -H "Content-Type: application/json" \
-  -d '{"jsonrpc": "2.0", "method": "eth_blockNumber", "params": [], "id": 1}'
+    -X POST \
+    -H "Content-Type: application/json" \
+    -d '{"jsonrpc":"2.0","method":"eth_blockNumber","params": [],"id":1}'
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx:11]:
```suggestion
    -d '{"jsonrpc": "2.0", "method": "eth_blockNumber", "params":  [], "id": 1}'
```

File: services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx [hunk 9]
```diff
@@ -1,15 +1,23 @@
...
+```
+
+  </TabItem>
+  <TabItem value="WSS">
+
+```bash
+wscat -c wss://zksync-mainnet.infura.io/ws/v3/YOUR-API-KEY -x '{"jsonrpc":"2.0","method":"eth_blockNumber","params": [],"id":1}'
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_blocknumber-request.mdx:18]:
```suggestion
wscat -c wss://zksync-mainnet.infura.io/ws/v3/YOUR-API-KEY -x '{"jsonrpc": "2.0", "method": "eth_blockNumber", "params": [], "id": 1}'
```

File: services/reference/zksync/json-rpc-methods/_eth_call-request.mdx [hunk 10]
```diff
@@ -1,15 +1,23 @@
-import Tabs from "@theme/Tabs"
-import TabItem from "@theme/TabItem"
+import Tabs from '@theme/Tabs';
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_call-request.mdx:1]:
```suggestion
import Tabs from "@theme/Tabs";
```

File: services/reference/zksync/json-rpc-methods/_eth_call-request.mdx [hunk 11]
```diff
@@ -1,15 +1,23 @@
...
+import TabItem from '@theme/TabItem';
```
Comment (joaniefromtheblock) [inline services/reference/zksync/json-rpc-methods/_eth_call-request.mdx:2]:
```suggestion
import TabItem from "@theme/TabItem";
```

File: services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx (hunk 2 above)
Comment (bgravenorst) [reply services/reference/zksync/json-rpc-methods/_eth_accounts-request.mdx]: Yep, I'll keep an eye out for that next time.

**Review by joaniefromtheblock — APPROVED** [review]
LGTM

**Review by joaniefromtheblock — APPROVED** [review]
LGTM

**Pull request merged**